packages/doenetml-worker-rust/lib-js-wasm-binding/src/math.ts

```diff
@@ -0,0 +1,521 @@
+    "arctan",
+    "arctanh",
+    "arg",
+    "min",
+    "max",
+    "mean",
+    "median",
+    "floor",
+    "ceil",
+    "round",
+    "sum",
+    "prod",
+    "variance",
+    "std",
+    "count",
+    "mod",
+    "Re",
+    "Im",
+    "det",
+    "trace",
+    "nPr",
+    "nCr",
+];
+
+let allowedLatexSymbols = [
+    "alpha",
+    "beta",
+    "gamma",
+    "Gamma",
+    "delta",
+    "Delta",
+    "epsilon",
+    "zeta",
+    "eta",
+    "theta",
+    "Theta",
+    "iota",
+    "kappa",
+    "lambda",
+    "Lambda",
+    "mu",
+    "nu",
+    "xi",
+    "Xi",
+    "pi",
+    "Pi",
+    "rho",
+    "sigma",
+    "Sigma",
+    "tau",
+    "Tau",
+    "upsilon",
+    "Upsilon",
+    "phi",
+    "Phi",
+    "chi",
+    "psi",
+    "Psi",
+    "omega",
+    "Omega",
+    "partial",
+    "varnothing",
+    "emptyset",
+    "angle",
+    "circ",
+    "$",
+    "%",
+];
+
+export var textToAst = new me.converters.textToAstObj({
+    appliedFunctionSymbols: appliedFunctionSymbolsDefault,
+});
+
+export function getTextToMathConverter({
+    functionSymbols,
+    appliedFunctionSymbols = appliedFunctionSymbolsDefault,
+    splitSymbols = true,
+    parseScientificNotation = false,
+}: {
+    functionSymbols?: string[];
+    appliedFunctionSymbols?: string[];
+    splitSymbols?: boolean;
+    parseScientificNotation?: boolean;
+} = {}) {
+    return (x: string) =>
+        me.fromAst(
+            new me.converters.textToAstObj({
+                appliedFunctionSymbols,
+                functionSymbols,
+                splitSymbols,
+                parseScientificNotation,
+            }).convert(x),
+        );
+}
+
+export var latexToAst = new me.converters.latexToAstObj({
+    appliedFunctionSymbols: appliedFunctionSymbolsDefaultLatex,
+    allowedLatexSymbols,
+});
+
+export function getLatexToMathConverter({
+    functionSymbols,
+    appliedFunctionSymbols = appliedFunctionSymbolsDefaultLatex,
+    splitSymbols = true,
+    parseScientificNotation = false,
+}: {
+    functionSymbols?: string[];
+    appliedFunctionSymbols?: string[];
+    splitSymbols?: boolean;
+    parseScientificNotation?: boolean;
+} = {}) {
+    if (splitSymbols) {
+        return (x: string) =>
+            me.fromAst(
+                new me.converters.latexToAstObj({
+                    appliedFunctionSymbols,
+                    functionSymbols,
+                    allowedLatexSymbols,
+                    parseScientificNotation,
+                }).convert(
+                    wrapWordIncludingNumberWithVar(x, parseScientificNotation),
+                ),
+            );
+    } else {
+        return (x: string) =>
+            me.fromAst(
+                new me.converters.latexToAstObj({
+                    appliedFunctionSymbols,
+                    functionSymbols,
+                    allowedLatexSymbols,
+                    parseScientificNotation,
+                }).convert(wrapWordWithVar(x, parseScientificNotation)),
+            );
+    }
```

Comment:
`let wrapper = splitSymbols ? wrapWordIncludingNumberWithVar : wrapWordWithVar` and then you don't need to duplicate all this code.

Reply:
Done